WordPress/Classes/Services/BlogService.m

```diff
@@ -159,9 +159,14 @@ - (void)syncBlogsForAccount:(WPAccount *)account
 {
     DDLogMethod();
 
+    NSManagedObjectID *accountID = account.objectID;
     id<AccountServiceRemote> remote = [self remoteForAccount:account];
     [remote getBlogsWithSuccess:^(NSArray *blogs) {
         [self.managedObjectContext performBlock:^{
+            WPAccount *account = (WPAccount *)[self.managedObjectContext existingObjectWithID:accountID error:nil];
+            if (!account) {
+                return;
```

Comment:
@astralbodies since I don't see many uses of `obtainPermanentIDForObject:` in the code, I'm guessing it's best to save the context before calling sync blogs, and assume service classes are passed objects that are already saved in the context?

We could refactor this further and update all the service classes to only take IDs, but again this PR was meant to just fix the crashes.